ironfish-cli/src/commands/wallet/multisig/commitment/create.ts

```diff
@@ -47,9 +47,12 @@ export class CreateSigningCommitmentCommand extends IronfishCommand {
 
     let identities = options.identity
     if (!identities || identities.length < 2) {
-      const input = await longPrompt('Enter the identities separated by commas', {
-        required: true,
-      })
+      const input = await longPrompt(
+        'Enter the identities of all participants who will sign the transaction separated by commas',
```

Comment:
(Not a native speaker/writer) Should there be a comma between "transaction" and "separated"?

Reply:
I think there should be. I had one in there but removed it because we had other uses without the comma 😅